modules/javafx.controls/src/test/java/test/javafx/scene/chart/AreaChartTest.java

```diff
@@ -497,6 +498,24 @@ public void testSeriesRemoveAnimatedStyleClasses() {
         assertArrayEquals(convertSeriesDataToPoint2D(expectedSeries).toArray(), findDataPointsFromPathLine(ac).toArray());
     }
 
+    //JDK-8283675
+    @Test public void testChartFillRemovedOnClearingSeries() {
```

Comment:
should we add a similar test for other chart types (even if those do not have the issue)?

Reply:
Could be useful. Do you think it should be done as part of this PR or should we create separate bug and take it up in test sprint?

Reply:
sure, we can use the upcoming test sprint to add these tests.  would you please create a ticket?

Reply:
Created [JDK-8314779](https://bugs.openjdk.org/browse/JDK-8283675)